cmd/root/root.go

```diff
@@ -75,12 +75,33 @@ var app = &cli.App{
 						return err
 					}
 
-					_, err = validator.Validate(data)
+					_, err = validator.ValidateRules(data)
 					if err != nil {
 						return cli.Exit(err, 1)
 					}
 				}
 
+				return nil
+			},
+		},
+		{
+			Name: "validate-config",
```

Comment:
Iiuc, "validate rules" works in tandem with "--format" (promql/logql).
Should we do the same for "validate config"?

Reply:
Yeah and since the Loki config is auto-generated by the charm, it should always be valid.